config/step/ot_ld_based_clumping.yaml

```diff
@@ -1,7 +1,7 @@
 defaults:
   - ld_based_clumping
 
+ld_index_path: ${datasets.ld_index}
 study_locus_input_path: ???
-ld_index_path: ???
```

Comment:
This path is always the same, added to main config.

Reply:
makes sense. Looks like a previous error